_scripts/instruction-widget/templates/install/epel.html

```diff
@@ -0,0 +1,64 @@
+
+<aside class="note">
+  <h4>Note:</h4>
+  {{#context.packaged}}
+    <p>
+      Certbot is packaged in EPEL (Extra Packages for Enterprise Linux). To use
+      Certbot, you must first <a
+      href="https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F">
+      enable the EPEL repository</a>.
+    </p>
+  {{/context.packaged}}
+  {{^context.packaged}}
+    <p>
+      Not all of Certbot's dependencies are available in the standard repositories.
+      To use Certbot, you must first <a
+      href="https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F">
+      enable the EPEL (Extra Packages for Enterprise Linux) repository</a>.
+    </p>
+  {{/context.packaged}}
+  {{#context.rhel}}
+    <p>
+      This can be done by downloading the proper epel rpm and installing it:
+      <pre>
+        $ rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{context.version}}.noarch.rpm
+      </pre>
+      Once you've done this, you can verify that it worked by running:
+      <pre>
+        $ yum repolist | grep epel
+      </pre>
+    </p>
+    {{#context.packaged}}
+      <p>
+        You must also enable the optional channels, which can be done by running these
+        commands:
+        <pre>
+          $ subscription-manager repos --enable rhel-7-server-extras-rpms
```

Comment:
I'm going off of: https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/rhsm/supplementary-repos

Unfortunately I can't read https://access.redhat.com/solutions/472163 because I don't have a redhat subscription